Added PR template (#1626)
huggingface/blog: .github/workflows/pull_request_template.md

# Blogs issue template
## Before submitting a blog post

> - Please make sure that you have communicated with Hugging Face staff and agreed that your blog post will be released on Hugging Face blog. 

Reviewer: ```suggestion
- Please make sure that you have communicated with Hugging Face staff and agreed that your blog post will be released on the Hugging Face blog. 
```

> - If this is not the case, don't fret! You can release your blog post as a Community Blog Post, that will be released on blog main page and it will show up on your profile too!

Reviewer: ```suggestion
- If this is not the case, don't fret! You can release your blog post as a Community Blog Post! It will be published on a special section in the main page of the Hugging Face blog, and it will show up on your profile too!
```

  To get started with writing a community blog post, simply join [this organization](https://huggingface.co/blog-explorers) and follow the instructions on organization card.

> ## Some guidelines

Reviewer: ```suggestion
## Important guidelines
```

- You can store your thumbnail in `/assets` and rest of the images on [our documentation-images repository](https://huggingface.co/datasets/huggingface/documentation-images) to keep this repository lightweight.

> - Every blog post has a metadata, please add this according to your blog, it looks like below:

Reviewer: ```suggestion
- Every blog post has metadata, please add according to your blog, like below:
```

```
---
title: "Open-Source Text Generation & LLM Ecosystem at Hugging Face"
thumbnail: /blog/assets/os_llms/thumbnail.png
authors:
- user: merve

> - guest: true

Reviewer: ```suggestion
  guest: true
```
